src/main/kotlin/com/pravo/pravo/domain/member/controller/MemberController.java

```diff
@@ -30,10 +31,11 @@ public MemberController(MemberService memberService) {
     }
 
     @PostMapping("/logout")
-    public ApiResponseDto<String> logoutPage(
+    public ApiResponseDto<Void> logoutPage(
         @RequestHeader("Authorization") String token
     ) {
-        return ApiResponseDto.success(memberService.logout(token));
+        memberService.logout(token);
+        return ApiResponseDto.success(null);
```

Comment:
```suggestion
        return ApiResponseDto.success(memberService.logout(token));
```
P2: void의 경우에도 다음처럼 반환하면 될 것 같습니다